src/ShareToken.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import firebase from './lib/firebase';
+import { db } from './lib/firebase';
+import { useCollection } from 'react-firebase-hooks/firestore';
+import { useHistory } from 'react-router-dom';
+import { withSnackbar } from 'notistack';
+
+const ShareToken = (props) => {
+  const [shareToken, setShareToken] = useState(null);
+
+  // 1. Make a on submit function where we pull list of collections
+  let history = useHistory();
+  const onSubmitHandler = (event) => {
+    event.preventDefault();
+    db.collection(shareToken)
+      .get()
+      .then((querySnapshot) => {
+        if (querySnapshot.empty) {
```

Comment:
I played around with this and found it interesting that Firebase doesn't throw an error if a collection doesn't exist, so if we do want to have the ability to add collections with empty lists, we'll probably want to change this logic in the future. However, this works fine for this update! Nice job 💯